docs/tutorials/11_working_with_classical_optimization_solvers.ipynb

```diff
@@ -0,0 +1,578 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with Classical Optimization Solvers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use classical optimization solvers (CPLEX and Gurobi) with Qiskit Optimization.\n",
+    "Docplex and Gurobipy are the Python APIs for CPLEX and Gurobi, respectively.\n",
+    "We can load and save an optimization model by Docplex and Gurobipy and can apply CPLEX and Gurobi to `QuadraticProgram`.\n",
+    "\n",
+    "If you want to use the CPLEX solver, you need to install `pip install 'qiskit-optimization[cplex]'`. Docplex is supposed to be installed with Qiskit Optimization.\n",
```

Comment:
Instead of

> Docplex is supposed to be installed with Qiskit Optimization.

how about

> Docplex is automatically installed, as a dependent, when you install Qiskit Optimization.